Adding information to README on use of Docker container

jrm16020 opened the pull request:
This includes a screenshot showing the use of the Docker container, fixes a bug in the tab completion that required a "-" before autocomplete would work, and also removes the dependency on the containerized Python backend.  Users can use the Docker Compose file provided in the HTTP Observatory repo to run the backend locally, or default to the public instance.

jrm16020 commented:
Addresses #22

gregglind commented:
You are pretty amazing!  Keep rocking this :)  Will roll this out with other 0.7 series changes right now :)

Commit:
Adding information to README on use of Docker container

Removing mod to shebang

This includes a screenshot showing the use of the Docker container, fixes a bug in the tab completion that required a "-" before autocomplete would work, and also removes the dependency on the containerized Python backend.  Users can use the Docker Compose file provided in the HTTP Observatory repo to run the backend locally, or default to the public instance.

jrm16020 commented:
Thank you @gregglind.
